cdap-docs/developers-manual/source/getting-started/building-apps.rst

```diff
@@ -93,44 +138,44 @@ Once an application is deployed:
     * - ``<program-id>``
-      - Name of the *Flow*, *MapReduce*, *Workflow*, or *Custom Service*
-        being called
+      - Name of the *adapter*, *flow*, *MapReduce*, *service*, *spark*, *worker* or *workflow* being called
+      
 
 Stopping an Application
-----------------------------------
+-----------------------
 
 Once an application is deployed:
 
-- On the Application's detail page in the CDAP UI, you can click on a program to go 
+- On an application's detail page in the CDAP UI, you can click on a program to go 
   to its detail page and then click the *Stop* button there; or
-- From the Standalone CDAP SDK directory, use the :ref:`Command Line Interface: <cli>`
-
-  .. list-table::
-    :widths: 20 80
-    :stub-columns: 1
-
-    * - On Linux:
-      - ``$ ./bin/cdap-cli.sh stop <program-type> <app-id.program-id>``
-    * - On Windows:
-      - ``> bin\cdap-cli.bat stop <program-type> <app-id.program-id>``
+- From the Standalone CDAP SDK directory, use the :ref:`Command Line Interface <cli>`::
 
+    $ cdap-cli.sh stop <program-type> <app-id.program-id>
+    
+    
 Removing an Application
-----------------------------------
+-----------------------
 
-Once an application is stopped |---| all Processes (Flows, MapReduce programs, Workflows,
-etc.), Queries, and Services are stopped |---| you can go to the *Management* page of the
-CDAP UI, click on the Namespaces > default > Applications to bring up all Applications, 
-and then click on a particular Application, and then from the Application's detail page,
-select the *Actions > Delete* menu in the upper right to delete the Application. 
+Once an application is stopped |---| all of its programs (flows, MapReduce programs,
+workflows, etc.), queries, and services are stopped |---| you can go to the |all_apps|_
+page of the CDAP UI, click on the particular application to go to its detail page, click
+the *Actions* menu on the right side and select *Manage* to go to the Management pane for
+the application, then click the *Actions* menu on the right side and select *Delete*.
 
 After confirmation, the application will be deleted.
 
-Note that any Storage (Datasets) created or used by the Application will remain, as they
-are independent of the Application. Datasets can be deleted with the 
-:ref:`HTTP Restful API <restful-api>`, the 
-:ref:`Java Client API <java-client-api>`, or the 
-:ref:`Command Line Interface API <cli>`.
+From the Standalone CDAP SDK directory, you can also use the Command Line Interface::
+
+    $ cdap-cli.sh delete app <app-id>
+
+Note that any storage (datasets) created or used by the application will remain, as they
+are independent of the application. Datasets can be deleted from the |datasets|_ page of
+the CDAP UI, or by using the :ref:`HTTP Restful API <restful-api>`, the 
+:ref:`Java Client API <java-client-api>`, or the :ref:`Command Line Interface API <cli>`.
+
+Streams cannot be deleted; instead, they can be truncated.
```

Comment:
Added an issue to update this for 3.1 docs: https://issues.cask.co/browse/CDAP-2554